init.lua

```diff
@@ -44,6 +44,15 @@ if minetest.get_modpath("toolranks") then
 end
 
 
+-- Pickaxe-Tweaks support
+
+if minetest.get_modpath("pick_axe_tweaks") and minetest.global_exists("pick_axe_tweaks") then
```

Comment:
```suggestion
if minetest.get_modpath("pick_axe_tweaks") then
```
Only really needs one check here.